templates/release/de/meter/cg-em24_1.yaml

```diff
@@ -1,6 +1,8 @@
 product:
   brand: Victron
   description: EM24
+description: |
+ EM24 mit RS-485 anschluss. Benutze die EM24_E1 für die EM24 mit Ethernet anschluss, denn die Definition is nicht compatibel.
```

Comment:
EM24 mit RS-485 Anschluss. Benutze die EM24_E1 für die EM24 mit Ethernet-Anschluss, denn die Definition is nicht kompatibel.